Sources/ShortcutRecorder/SRRecorderControl.m

```diff
@@ -1511,9 +1506,6 @@ - (void)viewWillMoveToWindow:(NSWindow *)aWindow
         [NSNotificationCenter.defaultCenter removeObserver:self
                                                       name:NSWindowDidResignKeyNotification
                                                     object:self.window];
-        [NSNotificationCenter.defaultCenter removeObserver:self
```

Comment:
I forget how all of this is handled now, but have you tested that dark mode works properly?